examples/chrono/chrono.cpp

```diff
@@ -0,0 +1,58 @@
+﻿/*
+ * Copyright (c) 2022, Christopher Kormanyos
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#include <chrono>
+
+#include <avr/io.h>
+
+auto main() -> int
+{
+  // Set the port pin value to low.
+  PORTB |= ~(1U << PORTB5);
```

Comment:
typo?
```suggestion
  PORTB |= (1U << PORTB5);
```
or 
```suggestion
  PORTB &= ~(1U << PORTB5);
```

Reply:
I will finish the draft tomorrow. Thanks. I will check it on a board and fix these typos.